code/web/src/modules/user/StyleSurvey.js

```diff
@@ -14,25 +15,73 @@ import { white, grey, grey2, grey3 } from "../../ui/common/colors";
 import { APP_URL } from "../../setup/config/env";
 import userRoutes from "../../setup/routes/user";
 
-const StyleSurvey = () => {
+const StyleSurvey = (props) => {
+  const styleDescriptions = {
+    classy:
+      "A Classy style Personality will always be elegantly and appropriately dressed for any occasion and always be well turned out.  Your style is simple, refined and somewhat restrained!",
+    nature:
+      "Feeling comfortable in your clothes is most important to you and your easygoing nature tends towards more casual outfits. Simple lines and designs are your preferred choices over anything too detailed or fussy. In keeping with this carefree attitude you tend to buy only easy, wash and wear garments. You prefer fabrics such as denim, cotton jersey and lightweight knits. Your jewellery choices reflect your minimal look and you tend to wear basic and durable accessories with footwear also focussed on comfort. ",
+    sporty:
+      "A Sporty style includes all forms of dressing with ease: from activewear to outdoorsy, sportswear to menswear. It’s about comfort; comfort from your clothes and with your body.",
+    punk: "this thing",
```

Comment:
If we had more time we could update the style description for punk (or maybe it's ironic?). But great job for getting the descriptions to display on the page!